fronted/src/components/PopupAuth.js

```diff
@@ -35,19 +34,32 @@ function PopupAuth(props) {
     });
   }
 
+  const submitSignupErrMessage = () => {
+    if (props.isUniqeMaillError) {
+     return 'This is email is already registered'
+    } else if (props.serverError) {
+      return 'Something went wrong with the server, please try again'
+    } else if (props.isIncorrectError) {
+      return 'Incorrect email or password'
+     } else {
+      return ''
+    }
+  }
+
+
   return (
-    <PopupWithForm isOpen={props.isOpen} onClose={props.onClose} title={props.title} handleSubmit={handleSubmit}>
+    <PopupWithForm isValidatedForm={props.isValidatedForm} isOpen={props.isOpen} onClose={props.onClose} title={props.title} handleSubmit={handleSubmit}>
       <label className='popup__input-title' >Email</label>
```

Comment:
**Might be better**

You need to format the code

you can do it with  `SHIFT + ALT + F` or  `prettier`

Reply:
I have installed pretiier and will activate it after the project will be accepted, just un case i will have some conflicts with eslint and more work to do.